_posts/2024-10-25-pycon-ug-2024.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Black Python Devs at PyconUG 2024"
+date: 2024-10-25
+author: "Tobias Thomas (Toe•bias HT)"
+lang: en
+featured_image: "/assets/images/pyconug.png"
```

Comment:
We have an image in canva for this sponsorship that should be used as the featured image.

link: https://www.canva.com/design/DAGQkaiQC88/G4IKEN9UrDbHEbm2DZBLQA/edit?utm_content=DAGQkaiQC88&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

@ToebiasHT - let me know if you can't download it and I will get it added to the images path.

Reply:
@kjaymiller I think I've been able to incorporate it. Shall I resolve the conversation?